packages/blaze-tools/tokens.js

```diff
@@ -22,7 +22,7 @@ var unicodeClass = function (abbrev) {
 // Match one or more characters that can start an identifier.
 // This is IdentifierStart+.
 var rIdentifierPrefix = new RegExp(
-  "^([a-zA-Z$_]+|\\\\u[0-9a-fA-F]{4}|" +
+  "^(@?[a-zA-Z$_]+|\\\\u[0-9a-fA-F]{4}|" +
```

Comment:
I think I will change it to just accept '@index' and nothing else starting with '@'